pax-web-itest/pax-web-itest-container/pax-web-itest-container-tomcat/src/test/java/org/ops4j/pax/web/itest/tomcat/WarJSFTCIntegrationTest.java

```diff
@@ -96,7 +92,7 @@ public void testSlash() throws Exception {
 		HttpTestClientFactory.createDefaultTestClient()
 				.withResponseAssertion("Response must contain 'Please enter your name'",
 						resp -> resp.contains("Please enter your name"))
-				.doGETandExecuteTest("http://127.0.0.1:8282/war-jsf-sample/");
+				.doGETandExecuteTest("http://127.0.0.1:8282/war-jsf-sample/index.jsp");
```

Comment:
Interesting, was this mandatory? 
Cause actually it should also work with index.jsp as the welcome files should take care of that. 
If this doesn't work we might also have an issue with welcome files in case of TC ...